classes/tweet.rb

```diff
@@ -1,4 +1,47 @@
 class Tweet
-  attr_accessor :content, :location, :created_at, :zombie, :tweet_id
+  attr_accessor :content, :location, :created_at,
+                        :zombie, :unique_id, :favs, :retweets,
+                        :original_tweet, :retweeted
+
+  # Class level attributes
+  @@tweet_id = 1
+  @@all_tweets = []
+
+  # Run this block on startup
+  def initialize
+    # Assign a unique id to each tweet and increment the counter
+    @unique_id = @@tweet_id
+    @@tweet_id += 1
+    # Create an empty array to track this tweets favs
+    @favs = []
+    # We do track who retweets this tweet, we just count them
+    @retweets = 0
+    # If this tweet is a retweet, the original tweet is stored here.
+    # Otherwise, this is left as false
+    @original_tweet = false
```

Comment:
While this will work, I will continue to reiterate that using the accessor methods are preferred, for if you mistype/misspell the method, the program will break (`undefined local variable or method`), where if you do the same with an instance variable, the program won't break, and you'll be confused.